### ISSUE TITLE
Storage solution

### ISSUE THREAD

**Issue by FrederikBolding** [Giveth/youtube-video-backup#2]
We probably need some sort of storage solution to save the backups onto, and a server or someone that runs the backup script at regular intervals.

**Comment by deamme** [comment]
I was thinking about using the IPFS endpoint of Giveth. cc @vojtechsimetka @perissology

**Comment by FrederikBolding** [comment]
@deamme - @geleeroyale was talking about using SyncThing - And I believe he's already set it up, and is getting people on board to host it, as well as figuring out hosting himself.

**Comment by vojtechsimetka** [comment]
@deamme @FrederikBolding I don't think that IPFS is a good solution. Storage is quite expensive and these videos are likely quite big.

**Comment by geleeroyale** [comment]
I think for now its perfectly acceptable to keep _at least one_ backup. I ran the script and plan on running it periodically, as well as set-up sharing on the folder via Syncthing. 

If somebody wants to help share the cloud, please PM me for access.

Right now the full backup clocks in at 55GB for 180 videos which is quite ok.
